.github/workflows/cdk-bootstrap.yml

```diff
@@ -70,18 +73,19 @@ jobs:
       - name: (CI) Configure AWS credentials
         uses: aws-actions/configure-aws-credentials@v1.6.1
         with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ap-southeast-2
           mask-aws-account-id: true
-          role-to-assume: ${{ secrets.AWS_ASSUME_ROLE_CI }}
+          role-to-assume: arn:aws:iam::586981104868:role/AWS_ASSUME_ROLE_CI_OIDC
```

Comment:
Role names are normally `CamelCase`. Also, the words "AWS" and "role" in the AWS role name are redundant. Maybe something like `OidcCi` or `CiOidc`?

Reply:
It's probably tidier ya. I copied the role from Github Secrets without much thought. Have fixed.